stdlib/multiprocessing/synchronize.pyi

```diff
@@ -30,7 +30,7 @@ class Condition(AbstractContextManager[bool]):
     ) -> None: ...
 
 class Event:
-    def __init__(self, lock: _LockLike | None = ..., *, ctx: BaseContext) -> None: ...
+    def __init__(self, *, ctx: BaseContext) -> None: ...
```

Comment:
It doesn't look like this parameter has ever existed at runtime (from what I can see); I'm not sure how it made its way into the stub